must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -108,25 +108,31 @@ for resource in "${resources[@]}"; do
     { oc adm --dest-dir="${BASE_COLLECTION_PATH}/namespaces/all/" inspect "${resource}" --all-namespaces --"${SINCE_TIME}"; } >> "${BASE_COLLECTION_PATH}/gather-debug.log" 2>&1
 done
 
-echo "collecting dump of openshift-dr-system namespace" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
-oc adm --dest-dir="${BASE_COLLECTION_PATH}" inspect ns/"${RAMEN_NAMESPACE}" --"${SINCE_TIME}" >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
-
-# Create the dir for oc_output for openshift-dr-system namespace
-mkdir -p "${BASE_COLLECTION_PATH}/namespaces/${RAMEN_NAMESPACE}/oc_output/"
-
-# Run the Collection of Resources to list
-for dr_resource in "${dr_resources[@]}"; do
-     echo "collecting oc command ${dr-resource}" | tee -a "${BASE_COLLECTION_PATH}/gather-debug.log"
-     COMMAND_OUTPUT_FILE=${BASE_COLLECTION_PATH}/namespaces/${RAMEN_NAMESPACE}/oc_output/${dr_resource// /_}
-     # shellcheck disable=SC2086
-     { oc get ${dr_resource} -n ${RAMEN_NAMESPACE}; } >> "${COMMAND_OUTPUT_FILE}"
-done
-
-# Run collection of oc describe command for config map
-echo "collecting oc describe configmap -n ${RAMEN_NAMESPACE}" | tee -a "${BASE_COLLECTION_PATH}/gather-debug.log"
+isRamenNamespace=$(timeout 60 oc get ns | grep openshift-dr-system | awk '{print $1}')
```

Comment:
You could also do `oc get ns openshift-dr-system` and verify the return code ($? == 0).